### PR TITLE
PLANNER-1986: [Tasks Assignment] Spring Boot Integration

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hi @rsynek , would you mind review this PR?

**Comment by rsynek** [comment]
jenkins execute springboot build

**Comment by wmedvede** [comment]
Hi @rsynek I was looking at the failing tests you'll see in the "Spring Boot" and I confirm that all this tests are also failing in master without any TaskAssiging spring boot related classes. So there must have been a change NOT related with this PR at some point, or this tests where never  executed since a long time and thus nobody noted it.
The task assigning integration tests has worked fine.

File: kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration-task-assigning-planning/src/main/java/org/kie/server/springboot/autoconfiguration/taskassigning/TaskAssigningPlanningKieServerAutoConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        initProperties();
+        return new TaskAssigningPlanningKieServerExtension();
+    }
+
+    @EventListener
+    public void handleContextRefreshEvent(ContextClosedEvent evt) {
+        restoreProperties();
+    }
+
+    private void initProperties() {
```
Comment (rsynek) [inline kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration-task-assigning-planning/src/main/java/org/kie/server/springboot/autoconfiguration/taskassigning/TaskAssigningPlanningKieServerAutoConfiguration.java:88]: I don't see the TaskAssigningConstants.TASK_ASSIGNING_RUNTIME_DELEGATE_PAGE_SIZE being used here. Is it intentional?

**Comment by wmedvede** [comment]
jenkins execute springboot build

File: kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration-task-assigning-planning/src/main/java/org/kie/server/springboot/autoconfiguration/taskassigning/TaskAssigningPlanningKieServerAutoConfiguration.java (hunk 1 above)
Comment (wmedvede) [reply kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration-task-assigning-planning/src/main/java/org/kie/server/springboot/autoconfiguration/taskassigning/TaskAssigningPlanningKieServerAutoConfiguration.java:89]: good catch, was missing. Added.

**Comment by wmedvede** [comment]
@rsynek fixes in place.

Spring boot tests passed.

The normal build failed in the following non related tests. I'll give it another try.

rg.kie.server.integrationtests.controller.WebSocketKieControllerRuleCapabilitiesIntegrationTest.testStartAndStopScanner[0: JSON KieServicesConfiguration{transport=REST, serverUrl='http://localhost:39504/kie-server-services/services/rest/server'}]

Failing for the past 1 build (Since Unstable#971 )
Took 5 min 38 sec.
Error Message
Synchronization failed for defined timeout: 240000 milliseconds.
Stacktrace
java.util.concurrent.TimeoutException: Synchronization failed for defined timeout: 240000 milliseconds.
Standard Output
18:19:22.385 [main] ERROR o.k.s.i.shared.KieServerDeployer - Cannot delete directory/home/jenkins/workspace/KIE/master/pullrequest/droolsjbpm-integration-pullrequests/droolsjbpm-integration/kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/target/kie-server-testing-server-local-repo
java.io.IOException: Unable to delete directory /home/jenkins/workspace/KIE/master/pullrequest/droolsjbpm-integration-pullrequests/droolsjbpm-integration/kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/target/kie-server-testing-server-local-repo/org/kie.



org.kie.server.integrationtests.controller.WebSocketKieControllerRuleCapabilitiesIntegrationTest.testScanNow[0: JSON KieServicesConfiguration{transport=REST, serverUrl='http://localhost:39504/kie-server-services/services/rest/server'}]

Failing for the past 1 build (Since Unstable#971 )
Took 4 min 36 sec.
Error Message
Synchronization failed for defined timeout: 240000 milliseconds.
Stacktrace
java.util.concurrent.TimeoutException: Synchronization failed for defined timeout: 240000 milliseconds.

**Comment by wmedvede** [comment]
jenkins retest this

**Pull request merged**